mk/Common.mk

```diff
@@ -232,7 +232,7 @@ endif
 ###############################################################################
```

Comment:
We could also update the default hash for x-dv, if we want.

Though, we don't _need_ to do that for every x-dv update, so it's fine to leave it for now.

![bilde](https://github.com/openhwgroup/core-v-verif/assets/75431602/981b3b76-ee3c-4a2b-983a-adeb88d524a7)

Reply:
How do we sequence this correctly? We first have to merge the cv32e40x-dv repo and then update this core-v-verif repo or is there another way.

Reply:
Alternative 1:
* Merge in arbitrary order.
* We can simply use `clonetb --x-main` to get the latest x-dv.

Alternative 2:
1. Merge the `x-dv` PR.
2. Update `VERIF_ENV_REF` in this `x/dev` PR.
    * (Note: Step 1 and 2 can be done in reverse order, since the currently latest commit on the x-dv PR will enter x-dv anyway. But that's an aside.)
3. Merge this PR in `x/dev`.

Reply:
I updated the VERIF_ENV_REF in this PR